apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -639,8 +666,30 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                     });
                 });
             } else {
-                metadataProp.options.map((ele: string) => {
-                    allowedList.push({ text: ele, value: ele });
+                metadataProp.options.map((ele: any) => {
```

Comment:
Refrain from using `any`.

Reply:
Keeping the data type as any as discussed offline as changing this might break existing components